charts/tempo-distributed/README.md

```diff
@@ -361,8 +361,8 @@ The memcached default args are removed and should be provided manually. The sett
 | distributor.tolerations | list | `[]` | Tolerations for distributor pods |
 | distributor.topologySpreadConstraints | string | Defaults to allow skew no more then 1 node per AZ | topologySpread for distributor pods. Passed through `tpl` and, thus, to be configured as string |
 | enterprise.enabled | bool | `false` |  |
-| enterprise.image.repository | string | `"grafana/enterprise-traces"` | Grafana Enterprise Metrics container image repository. Note: for Grafana Tempo use the value 'image.repository' |
-| enterprise.image.tag | string | `"v2.6.1"` | Grafana Enterprise Metrics container image tag. Note: for Grafana Tempo use the value 'image.tag' |
+| enterprise.image.repository | string | `"grafana/enterprise-traces"` | Grafana Enterprise Tetrics container image repository. Note: for Grafana Tempo use the value 'image.repository' |
```

Comment:
I think this was meant to be `Traces` not `Tetrics`.

Reply:
Oh my bad.
Good catch.